secure-mobile-push/smp-android-client/securemobilepush/src/main/java/de/adorsys/android/securemobilepush/sms/SmsTool.java

```diff
@@ -0,0 +1,33 @@
+package de.adorsys.android.securemobilepush.sms;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.support.annotation.RequiresApi;
+import android.support.v4.content.ContextCompat;
+
+public class SmsTool {
```

Comment:
Fixed in commit: https://github.com/adorsys/secure-key-storage/commit/761c856002a1a3f6393ea09c990158803e09aa51